corehq/apps/reports/models.py

```diff
@@ -268,7 +269,11 @@ def _dispatcher(self):
             if dispatcher.prefix == self.report_type:
                 return dispatcher()
 
-        raise Exception("Unknown dispatcher: %s" % self.report_type)
+        logging.error("This saved-report (id: %s) is unknown (report_type: %s). (Might be a legacy report)" % (
```

Comment:
The exception is caught as general exception, as there are multiple places I didn't replace them all - and so couldn't find a better place. Should I change this to `notify_error` instead?